### ISSUE TITLE
Error in request-digest.es5.js isn't checked before using response

### ISSUE THREAD

**Issue by somec** [bnjjj/node-request-digest#13]
In file request-digest.es5.js, around line 123, version 1.0.9:

```
return (0, _request2['default'])(options, function (error, response, body) {
    if (response.statusCode >= 400) {
        var errorMessage = {
            statusCode: response.statusCode,
            response: response,
            body: body
        };

        return callback(errorMessage);
    }
    callback(error, response, body);
});
```

It should test if **error** is set before trying to access **response**, since if there is an error, response might not be set.

A minimal change is to alter the if-statement to:

`if (!error && response.statusCode >= 400) {`

**Comment by bnjjj** [comment]
Thanks a lot, I'll check that this weekend, don't hesitate to remind me if it's not done :)

**Comment by Aigelov** [comment]
Hi :)

When will be this issue fixed?

**Comment by bnjjj** [comment]
Done and published in 1.0.10 :) Thanks for your feedback
